index.html

```diff
@@ -0,0 +1,396 @@
+                                    <p class="match-date">25/10/2016 | 21:30</p>
+                                    <img class="img-responsive" src="images/logo/20462440.png" alt="Barça">
+                                    <span>VS</span>
+                                    <img class="img-responsive" src="images/logo/2761733.png" alt="Espanyol">
+                                    <div class="clubs-title">
+                                        <p class="home-match">FC Barcelona</p>
+                                        <p class="away-match">Espanyol</p>
+                                    </div>
+                                </div>
+                                <div class="match-main">
+                                    <h3 class="match-main-title">Champions League | Group stage - C</h3>
+                                    <p class="match-date">1/11/2016 | 20:45</p>
+                                    <img class="img-responsive" src="images/logo/30624218.png" alt="manCity">
+                                    <span>VS</span>
+                                    <img class="img-responsive" src="images/logo/20462440.png" alt="Barça">
+                                    <div class="clubs-title">
+                                        <p class="home-match">ManCHester City</p>
+                                        <p class="away-match">FC Barcelona</p>
+                                    </div>
+                                </div>
+                                <div class="match-main">
+                                    <h3 class="match-main-title">La Liga | Match day 14</h3>
+                                    <p class="match-date">3/12/2016 | 16:15</p>
+                                    <img class="img-responsive" src="images/logo/20462440.png" alt="Barça">
+                                    <span>VS</span>
+                                    <img class="img-responsive" src="images/logo/2761837.png" alt="Madrid">
+                                    <div class="clubs-title">
+                                        <p class="home-match">FC Barcelona</p>
+                                        <p class="away-match">Real Madrid</p>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+            <!--    End schedule       -->
+            <!--    Start Main Carousel      -->
+        <div id="carousel-example-generic" class="carousel slide main-carousel" data-ride="carousel">
+          <!-- Wrapper for slides -->
+          <div class="carousel-inner" role="listbox">
+            <div class="item active">
+              <img src="images/news/29252139.jpg" alt="messi">
+              <div class="carousel-caption main-carousel-caption hidden-xs">
+                <h1>FC Barcelona Get Straight Sack To Training</h1>
```

Comment:
There shouldn't be too many H1 elements in a single page, typically a page should only have a single H1, a few H2's, and a bunch of H3's.

In H*x*, think of the *x* as the number of permitted occurrences of the heading type.